app/lib/backend/errors.rb

```diff
@@ -13,7 +13,7 @@ module Backend
     'provider.invalid_transaction_id' => 'transaction does not exist'
   }.freeze
 
-  class Error < StandardError
+  class Errors < StandardError
```

Comment:
This class name is weird, I think I'd prefer to rename the file.

Reply:
True. Done.